source/guide_jellyfin.rst

```diff
@@ -0,0 +1,176 @@
+.. highlight:: console
+
+.. author:: knhash <https://knhash.in>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: streaming
+.. tag:: mediaplayer
+.. tag:: gallery
+
+
+.. sidebar:: About
+
+  .. image:: _static/images/jellyfin.svg
+      :align: center
+
+##########
+Jellyfin
+##########
+
+.. tag_list::
+
+Jellyfin is the volunteer-built media solution that puts you in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.
+
+You can use Jellyfin alongside :lab:`Syncthing <guide_syncthing>`, to load your media onto Uberspace for streaming.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  - :manual:`supervisord <daemons-supervisord>`
+  - :manual:`domains <web-domains>`
+
+License
+=======
+
+The software is licensed under `GNU General Public License v2.0`_. All relevant information can be found in the GitHub_ repository of the project.
+
+Prerequisites
+=============
+
+Your dashboard URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Create a directory in ``/home/isabell`` for jellyfin and its files.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ mkdir ~/jellyfin
+  [isabell@stardust ~]$
+
+Download the combined archive from release_, including both the server and WebUI, as we will need both of them.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ wget -O ~/jellyfin/jellyfin.tar.gz https://repo.jellyfin.org/releases/server/linux/stable/combined/jellyfin_10.7.7_amd64.tar.gz
+  [isabell@stardust ~]$
+
+Extract the archive. Delete the archive post extraction.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ tar -xvzf ~/jellyfin/jellyfin.tar.gz  -C jellyfin
+  [isabell@stardust ~]$ rm ~/jellyfin/jellyfin.tar.gz
+  [isabell@stardust ~]$
+
+Create a symbolic link to the Jellyfin 10.7.7 directory. This allows an upgrade by repeating the above steps and enabling it by simply re-creating the symbolic link to the new version.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ ln -s ~/jellyfin/jellyfin_10.7.7 ~/jellyfin/jellyfin
+  [isabell@stardust ~]$
+
+Create four sub-directories for Jellyfin data, and one to store your media library.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$  mkdir data cache config log library
```

Comment:
Please use absolute paths. I think that should be `~/jellyfin/data` and so on?

Reply:
Ack, fixed now.